httpcore/__init__.py

```diff
@@ -11,6 +11,7 @@
     ProxyError,
     ReadError,
     ReadTimeout,
+    TimeoutException,
```

Comment:
Would need to add it to `__all__` too. :)

Reply:
done, sorry :smile: